tests/api.c

```diff
@@ -19884,6 +19884,8 @@ static int test_wc_i2d_PKCS12(void)
     return EXPECT_RESULT();
 }
 
+#ifndef __WATCOMC__
+/* There are known Watcom incompatibilities in this implementation  */
```

Comment:
How did this off-topic Watcom compiler stuff sneak into this PR?  Seems like we should fix this with higher priority in a separate PR.

Reply:
Upon reconsideration, it was inappropriate for me to include the Watcom gate-out to avoid the error. 

I've excluded that update from the most recent commit / squash in this PR.

I still plan to create a new PR to address the watcom compiler issue, including a new test since it is not detected by any current tests.